packages/manager/src/features/Linodes/LinodesDetail/LinodeConfigs/utilities.test.ts

```diff
@@ -34,3 +42,27 @@ describe('getPrimaryInterfaceIndex', () => {
     expect(getPrimaryInterfaceIndex(interfaces)).toBe(null);
   });
 });
+
+describe('useGetDeviceLimit', () => {
+  it('should always return 8 as the device limit', () => {
+    queryMocks.useFlags.mockReturnValue({
+      blockStorageVolumeLimit: false,
+    });
+
+    expect(useGetDeviceLimit(131072)).toEqual(8);
+    expect(useGetDeviceLimit(65536)).toEqual(8);
+    expect(useGetDeviceLimit(16384)).toEqual(8);
+    expect(useGetDeviceLimit(1024)).toEqual(8);
+  });
```

Comment:
We may want to treat `useGetDeviceLimit` as a hook in our unit testing for correctness.

```suggestion
  it.each([131072, 65536, 16384, 1024])(
    'should always return 8 as the device limit',
    (value) => {
      const { result } = renderHook(() => useGetDeviceLimit(value), {
        wrapper: (ui) =>
          wrapWithTheme(ui.children, {
            flags: { blockStorageVolumeLimit: false },
          }),
      });

      expect(result.current).toEqual(8);
    }
  );
```

As a bonus it lets us avoid mocking `src/hooks/useFlags`

Reply:
done, thank you!!